bundles/scripting/sightly/java-compiler/src/main/java/org/apache/sling/scripting/sightly/render/AbstractRuntimeObjectModel.java

```diff
@@ -67,6 +67,16 @@ public boolean isPrimitive(Object obj) {
     }
 
     @Override
+    public boolean isDate(Object target) {
+        return (target instanceof Date);
```

Comment:
As a developer you don't want to worry about this, you have a 'date' object. Whether this is a Date or Calendar.
Would be great if this can be implemented.
You can have a look at my implementation, this supports date as well as calendar.